deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/conf/ConvolutionMode.java

```diff
@@ -60,6 +60,6 @@
  */
 public enum ConvolutionMode {
 
-    Strict, Truncate, Same
+    Strict, Truncate, Same, Valid
```

Comment:
Why add valid? and how does it differ from truncate (unless it's an alias)?
If it is necessary (or just an alias) - update the javadoc.